### PR TITLE
Ellie connect to backend

### PR DESCRIPTION
**Pull request by EllieChen-Git**
Please Nina please please please 

I love you

### CODE REVIEW

File: src/components/forms/SignInForm.js [hunk 1]
```diff
@@ -1,55 +1,82 @@
...
+    //   .then(response => {
+    //     console.log(response);
+    //     this.props.setAuthToken(response.data);
+    //     this.props.history.push("/");
+    //   })
+    //   .catch(error => {
+    //     console.log(error);
+    //     console.log("not working");
+    //     this.setState({ error });
+    //   });
```
Comment (INRokh) [inline src/components/forms/SignInForm.js:26]: I think it is safe to remove this code now.

**Review by INRokh — APPROVED** [review]
Great job! Thanks!

**Comment by INRokh** [comment]
I'll go ahead and merge.

File: src/components/forms/SignInForm.js (hunk 1 above)
Comment (EllieChen-Git) [reply src/components/forms/SignInForm.js:26]: OK! Thanks :)

**Pull request merged**